web/README.md

```diff
@@ -12,6 +18,22 @@ Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 The page will reload if you make edits.<br />
 You will also see any lint errors in the console.
 
+### `yarn lint`
```

Comment:
Can you mention in the desc about they're based on gts?

Reply:
Done!